samples/createTask.js

```diff
@@ -65,14 +65,10 @@ async function createTask(
     };
   }
 
-  const request = {
-    parent: parent,
-    task: task,
-  };
-
   console.log('Sending task:');
   console.log(task);
   // Send create task request.
+  const request = {parent, task};
```

Comment:
I think this breaks the sample: request.parent and request.task are mandatory, the previous version was correct.

Reply:
This is [javascript short hand notation for objects](https://alligator.io/js/object-property-shorthand-es6/). The variables must be named `parent` and `task` to work. This expanded would look like:
```
 const request = {	
    parent: parent,	
    task: task,	
 };
```

Reply:
Ah I see, thanks!